pagopa/io-app
# fix(IDPay): [IODPAY-147,IODPAY-135,IODPAY-128] Refactor list item design

## Short description
refactor of `TimelineListItem.tsx` to follow its new design

## List of changes proposed in this pull request
<img src="https://user-images.githubusercontent.com/60693085/217863591-03b2fe5a-5126-4a56-97e4-97bf4edaa920.png" width=200/>

- addition of new icon, as per design 
- addition of possible `greenLight` color to the `H4` component
- rework of the timelineListItem filtering system to allow for more variety
- the component now more strictly adheres to the new design
- update of Wallet and Timeline definitions
   - update of required tests
   - new map to generate the correct SVG icons
   - required update of timeline saga and `isLastPage` selector

DEPENDS ON https://github.com/pagopa/io-app/pull/4362

## How to test
Having one fully configured initiative with various timeline events, navigate to the initiative details' page, possibly also to the "show more" section, and verify that all the listItems render correctly.
